From: Torsten Werner <twerner@debian.org>
Date: Tue, 15 Jun 2010 23:08:01 +0200
Subject: [PATCH] disable tests for entitymanager

The following B-D are missing:

org.jboss.shrinkwrap:shrinkwrap-api
org.jboss.shrinkwrap:shrinkwrap-impl-base
org.hibernate:hibernate-jpamodelgen
---
 entitymanager/pom.xml |   14 ++++++++++++++
 1 files changed, 14 insertions(+), 0 deletions(-)

--- a/project/hibernate-entitymanager/pom.xml
+++ b/project/hibernate-entitymanager/pom.xml
@@ -74,6 +74,7 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+<!-- missing in Debian
         <dependency>
             <groupId>org.jboss.shrinkwrap</groupId>
             <artifactId>shrinkwrap-api</artifactId>
@@ -87,24 +88,29 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+-->
             <!--
                 Only really needed for the antrun plugin defined below (which in turn is only really needed
                 to work around a missing feature on the compile plugin; lovely no?).  However I have not
                 found a way to declare this dependency local to the antrun plugin and then reference it
                 from the <javac />'s <classpath /> :(
             -->
+<!-- missing in Debian
             <groupId>${project.groupId}</groupId>
             <artifactId>hibernate-jpamodelgen</artifactId>
             <version>1.0.0.Final</version>
             <scope>test</scope>
             <exclusions>
                 <exclusion>
+-->
                     <!-- Use the jpa-api defined here, crossing fingers they match up signature-wise -->
+<!-- missing in Debian
                     <groupId>org.hibernate.javax.persistence</groupId>
                     <artifactId>hibernate-jpa-2.0-api</artifactId>
                 </exclusion>
             </exclusions>
         </dependency>
+-->
     </dependencies>
 
     <build>
@@ -137,10 +143,13 @@
                     </bytecodeInjections>
                 </configuration>
             </plugin>
+<!--
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-antrun-plugin</artifactId>
+-->
                 <!-- Define one configuration outside executions and without a phase in order to allow 'mvn antrun:run' -->
+<!--
                 <configuration>
                     <tasks>
                         <property name="target.dir" value="${project.build.directory}/generated-src/jpamodelgen"/>
@@ -157,7 +166,9 @@
                         <id>process_annotations</id>
                         <phase>process-test-sources</phase>
                         <configuration>
+-->
                             <!-- No additional configuration needed. Unbound configuration gets inherited. (Feels wrong, but that's how it is) -->
+<!--
                             <testSourceRoot>${project.build.directory}/generated-src/jpamodelgen</testSourceRoot>
                         </configuration>
                         <goals>
@@ -166,10 +177,12 @@
                     </execution>
                 </executions>
             </plugin>
+-->
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
+                    <skip>true</skip>
                     <source>1.5</source>
                     <target>1.5</target>
                 </configuration>
@@ -178,6 +191,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
+                    <skipTests>true</skipTests>
                     <excludes>
                         <exclude>**/*TestCase.java</exclude>
                     </excludes>
